tests/containers/search/filters/agencyFilter/AgencyListContainer-test.jsx

```diff
@@ -149,6 +147,18 @@ describe('AgencyListContainer', () => {
             expect(container.state().autocompleteAgencies[2].title).toEqual('DEF Agency (DEF)');
             expect(container.state().autocompleteAgencies[2].subtitle).toEqual('Sub-Agency of Department ABC (ABC)');
         });
+        it('should not contain agency abbreviations', async () => {
+            const container = setupShallow(initialFilters);
+            container.setState({
+                agencySearchString: 'abc'
+            });
+
+            container.instance().parseAutocompleteAgencies(mockNullAgencyResults);
+
+            // Toptier agencies should have been moved to the top of the list
```

Comment:
Let's also update this comment which is a duplicate of `line 144`

Reply:
oops